src/main/java/seedu/address/model/contact/Contact.java

```diff
@@ -55,6 +60,8 @@ public Email getEmail() {
         return email;
     }
 
+    public ProfilePicture getProfilePicture() {return profilePicture; }
```

Comment:
I think a line break is needed here

Reply:
no need for a linebreak, following convention from existing code